plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/elastic/ElasticAgentExtension.java

```diff
@@ -132,12 +133,26 @@ public String getStatusReport(String pluginId) {
             throw new UnsupportedOperationException(format("Plugin `%s` implements Elastic Agent V1, `StatusReport` endpoint is not supported in this version.", pluginId));
```

Comment:
Ok, I think we should also add check for plugin capabilities after version check